cl_dll/discord_integration.cpp

```diff
@@ -24,8 +26,26 @@ namespace discord_integration
 		// This seems to be consistent across PCs.
 		constexpr const char STEAM_APP_ID[] = "17215498729465839686";
 
+		struct case_insensitive_hash
```

Comment:
Why add this much more complex solution instead of what was already here? Just store the original map_name in a variable before converting to lowercase for the rest of the code.

Reply:
Isn't that clear? Imagine that in the list, for example, a contributor writes the name of the map, say, `aGtRiCkS`, when in fact you have an `agtricks` map loaded in the game, and what will you do next? Should we manually change the list every time if that kind of thing happen? That seems like too much to do it each time, so that's that the reason why is such complex solution here

Reply:
`// Maps in this list must be lowercase`, then I look at it during review and ask to change

Reply:
Uhhhhh... Well, at least in your defense, can be said that selecting entire list and converting it to lower case in IDE shouldn't take than a few seconds to do, so it’s might be really not worth to going for such complex solution